utils/cliutils/utils.go

```diff
@@ -68,16 +69,25 @@ func ExitOnErr(err error) {
 	}
 }
 
-func FailNoOp(err error, success, failed int, failNoOp bool) error {
-	if exitCode := GetExitCode(err, success, failed, failNoOp); exitCode == ExitCodeFailNoOp {
-		return CliError{exitCode, "No files were affected."}
+func CheckForErrorResults(err error, success, failed int, failNoOp bool) error {
```

Comment:
I suggest renaming this function to **GetCliError**.

Reply:
Fixed.